applications/ConstitutiveModelsApplication/CMakeLists.txt

```diff
@@ -111,4 +111,4 @@ install(TARGETS KratosConstitutiveModelsCore DESTINATION libs )
 install(TARGETS KratosConstitutiveModelsApplication DESTINATION libs )
 
 # Add to the KratosMultiphisics Python module
-install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/ConstitutiveModelsApplication.py" DESTINATION KratosMultiphysics )
+install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/python_scripts/__init__.py" DESTINATION "KratosMultiphysics/ConstitutiveModelsApplication")
```

Comment:
this was changed again (minor change only), see the updated wiki:
https://github.com/KratosMultiphysics/Kratos/wiki/Applications-as-python-modules

sorry for the inconvenience

Reply:
Ok I update the change.